docs/host/host.md

```diff
@@ -80,3 +80,81 @@ If you are testing Harvester in a QEMU environment, you'll need to use QEMU v6.0
 
 ![Edit Config](/img/v1.1/host/edit-config.png)
 ![Add Disks](/img/v1.1/host/add-disks.png)
+
+## Ksmtuned Mode
+
+Ksmtuned is a KSM automation tool deployed as a DaemonSet to run Ksmtuned per node and start or stop KSM by setting the available memory percentage ratio (**Threshold Coefficient**), By default, you need to manually enable Ksmtuned on each node UI. You can see the KSM statistics from the node UI ([see the KSM in detail](https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html#ksm-daemon-sysfs-interface))
+
+### **Quick Run**
+
+1. Go to the **Hosts** page.
+2. On the node you want to modify, click **⋮ > Edit Config**.
+3. Select the **Ksmtuned** tab and **Run Strategy** in Run.
+4. (Optional)You can change **Threshold Coefficient** to match your expectations.
+
+![Edit Ksmtuned](/img/v1.1/host/edit-ksmtuned.png)
+
+5. Click **Save** to update.
+6. Wait at least 2 minutes and check the **Statistics**, click **⋮ >  Your Node > Ksmtuned tab.**
+
+![View Ksmtuned Statistics](/img/v1.1/host/view-ksmtuned-statistics.png)
+
+### **Parameters**
+
+**Run Strategy：**
+
+- **Stop:** Stop Ksmtuned and KSM，VM can still use shared memory pages.
+- **Run:** Run Ksmtuned。
+- **Prune:** Stop Ksmtuned and prune KSM memory pages.
+
+**Threshold Coefficient**:  configures the available memory percentage ratio, if the available memory is less than the threshold starts KSM, otherwise, it stops KSM.
```

Comment:
```suggestion
**Threshold Coefficient**: configures the available memory percentage ratio. If the available memory is less than the threshold, KSM will be started; otherwise, KSM will be stopped.
```

Reply:
If KSM is started/stopped automatically, then I think using passive voice is better